### DISCUSSION TITLE
How to save exceptions and dictionary words in user settings?

### DISCUSSION THREAD

**Discussion by bdmariobd** [valentjn/vscode-ltex#445] [Q&A] [answered]
Can I save my 'exceptions' and dictionary words to user settings by default instead of saving into .vscode settings folder?
I mean: those options -> user settings.json
![image](https://user-images.githubusercontent.com/56127361/139538956-f1002751-d3cf-4c2a-ad4a-ad939485c083.png)

**Comment by valentjn** [comment, marked as answer]
Yes, set [`ltex.configurationTarget`](https://valentjn.github.io/ltex/settings.html#ltexconfigurationtarget) as follows:

```json
{
  "ltex.configurationTarget": {
    "dictionary": "user",
    "disabledRules": "user",
    "hiddenFalsePositives": "user"
  }
}
```
